sites/upsun/src/development/troubleshoot.md

```diff
@@ -118,26 +118,75 @@ If you are using SSH, see how to [troubleshoot SSH access](../development/ssh/tr
 
 ## HTTP responses 502 Bad Gateway or 503 Service Unavailable
 
-If you see these errors when accessing your application,
-it indicates your application is crashing or unavailable.
-
-Typical causes and potential solutions include:
-
-- Your app is listening at the wrong place.
-  - Check your app's [upstream properties](/create-apps/app-reference/single-runtime-image.md#upstream).
-  - If your app listening at a port, make sure it's using the [`PORT` environment variable](/development/variables/use-variables.md#use-provided-variables).
-- Your `{{< vendor/configfile "app" >}}` configuration has an error and a process isn't starting
-  or requests can't be forwarded to it correctly.
-  - Check your `web.commands.start` entry or your `passthru` configuration.
-- The amount of traffic coming to your site exceeds the processing power of your application.
-  - You may want to [check if bots are overwhelming your site](https://support.platform.sh/hc/en-us/community/posts/16439634723858).
-- Certain code paths in your application are too slow and timing out.
-  - Check your code is running smoothly.
-  - Consider using the [observability solution](../increase-observability/application-metrics/_index.md) included in your project to get a better view of your application.
-- A PHP process is crashing because of a segmentation fault.
-  - See [how to deal with crashed processes](../languages/php/troubleshoot.md#troubleshoot-a-crashed-php-process).
-- A PHP process is killed by the kernel out-of-memory killer.
-  - See [how to deal with killed processes](../languages/php/troubleshoot.md#troubleshoot-a-killed-php-process).
+If you encounter a **502 Bad Gateway** or **503 Service Unavailable** error while accessing your application, it typically indicates that your application is either crashing or unavailable. Below are the typical causes and potential solutions:
+
+#### 1. Your app is listening at the wrong place
+- **Solution**: Check your app's [upstream properties](/create-apps/app-reference/single-runtime-image.md#upstream).
+- If your app is listening at a specific port, verify that it’s using the [`PORT` environment variable](/development/variables/use-variables.md#use-provided-variables) and ensure that it is listening at the correct port. 
+
+#### 2. Configuration issues in `.upsun/config.yaml`
+- **Solution**: Review your `.upsun/config.yaml` configuration.
+- Look for any errors that might prevent a process from starting or cause issues with requests being forwarded properly.
+- Check the `web.commands.start` entry or your `passthru` configuration for any misconfigurations.
```

Comment:
I would propose to point our customers to the logs of their apps in order to understand what's going wrong here as well.

Indeed, one of the possible issues here is that they haven't configured their `web.commands.start` correctly, but identifying potential typos or what's really going south is only possible when they look into their logs.